advanced/am-pattern-1/confs/am/instance-2/repository/conf/deployment.toml

```diff
@@ -231,5 +94,5 @@ subscriptions=["POST_ADD_USER"]
 [service_provider]
 sp_name_regex = "^[\\sa-zA-Z0-9._-]*$"
 
-#[transport.passthru_https.sender.parameters]
-#HostnameVerifier = "AllowAll"
+[database.local]
+url = "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE"
```

Comment:
@Chathuru add an EOF line.

Reply:
Fix the issue in https://github.com/wso2/kubernetes-apim/pull/344/commits/c1d371588b9d9e50fa267a4bbf5a74bc7b50517b